packages/design-system/src/__theme__/default/index.css

```diff
@@ -13,12 +13,15 @@
   /* Naming convention: 
     *
     * (Atom-Category-Component-Variant-Surface-State-PresenationalAttribute)
-    * Atom - Type of token (colors).
-    * Category - Indicates which category the token belongs to (action, response, control etc).
+    * Atom - Type of token (color/spacing/typography).
+    * Category - Indicates which category the token belongs to (action, response, control, content).
     * Component - Indicates which component the token belongs to (button, input, select etc).
-    *             this should be avoided as much as possible.
+    *             This is optional if the token is not component specific. For example, Avatar component
```

Comment:
Oh this is perfect, can we just add this on the avatar and tab components as well? Or maybe link this there.